src/main/java/com/spartronics4915/frc2020/Constants.java

```diff
@@ -1,4 +1,7 @@
 package com.spartronics4915.frc2020;
 
 public final class Constants {
+    public static final int kIndexerSpinnerId = -1;
```

Comment:
Our new convention is to have all subsystem variables be in their own subclass, so it'll look like this (and `Indexer` is stricken from the name): 

```
public final class Constants {
    public static final class Indexer { 
        public static final int kSpinnerId = -1;
         ... other variables
    }
    // ... other subclasses
}
```